app/models/access_token.rb

```diff
@@ -132,6 +133,12 @@ def scopes=(values)
     super Array(values).select(&:present?)
   end
 
+  def validate_scope_exists
+    scopes_allowed_values = self.class.allowed_scopes.values
```

Comment:
I think this will depends on the user permissions as well

If an admin gives a member `account_management` only, it should not be able to create an access token for `finance`

Reply:
We are validating those things in authentication when the access token is used. Should it be added here too? Right now this works just like it currently does for the UI, accepting any of the allowed ones.

Reply:
Right would be nice to make a comment in the code why we allow that

Reply:
This is showed a bit in https://github.com/3scale/porta/pull/430#discussion_r242837137
And the UI:
![image](https://user-images.githubusercontent.com/11318903/50217294-957ddc80-0388-11e9-9fec-cf73bbd42ca6.png)

Reply:
The UI is shown with:
https://github.com/3scale/porta/blob/6d7ee3c6ce333a197b41ecec89af64c2da2867a2/app/views/provider/admin/user/access_tokens/_form.html.slim#L5-L7

Reply:
Yes, that is what I showed also in https://github.com/3scale/porta/pull/430#discussion_r242837137 😄